packages/xstate-wallet/src/channel-wallet.ts

```diff
@@ -0,0 +1,65 @@
+import {Store} from './store/memory-store';
+import {MessagingServiceInterface} from './messaging';
+
+import {applicationWorkflow} from './workflows/application';
+import ReactDOM from 'react-dom';
+import React from 'react';
+import WalletUi from './ui/wallet';
+import {interpret, Interpreter} from 'xstate';
+import {Guid} from 'guid-typescript';
+import {convertToOpenEvent} from './utils/workflow-utils';
+export interface Workflow {
+  id: string;
+  machine: Interpreter<any, any, any>;
+  domain: string; // TODO: Is this useful?
+}
+export class ChannelWallet {
+  private workflows: Workflow[];
+
+  constructor(private store: Store, private messagingService: MessagingServiceInterface) {
```

Comment:
Might be worth initializing these inside, instead of passing them in (especially the MessagingService, which there will probably only be one version of).

Reply:
I'm going to leave as is to allow for dependency injection for now.